src/main/java/pl/javastart/task/Televisor.java

```diff
@@ -0,0 +1,29 @@
+package pl.javastart.task;
+
+class Televisor {
+    private boolean isOn;
+
+    public Televisor(boolean isOn) {
+        this.isOn = isOn;
+    }
+
+    public boolean isOn() {
+        return isOn;
+    }
+
+    public void setOn(boolean on) {
+        isOn = on;
+    }
```

Comment:
Jeżeli tworzysz klasy, w których jest jakaś logika, czyli metody pozwalające włączać, czy wyłączać telewizor, to raczej w takiej sytuacji nie dodajemy getterów i setterów.
W Javie możemy sobie wyróżnić dwa rodzaje klas reprezentujących dane:
- anemiczny model danych - czyli klasy, które reprezentują dane i jedyny sposób interakcji z nimi to właśnie set/get
- "bogaty" model domenowy - czyli klasy, w których używamy enkapsulacji. Stanem obiektu manipulujemy przy pomocy różnych metod, a wszystkie pola są prywatne i nie da się ich tak po prostu przestawić setterem

Te dwie metody możesz więc spokojnie usunąć, bo poniższe trzy metody pozwalają Ci zrealizować wszystkie założenia zadania.